src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -126,16 +132,28 @@ async def _handle_unary_unary_rpc(object method_handler,
     )
 
     # Sends response message
-    cdef tuple send_ops = (
-        SendStatusFromServerOperation(
-            tuple(),
-            StatusCode.ok,
-            b'',
-            _EMPTY_FLAGS,
-        ),
-        SendInitialMetadataOperation(None, _EMPTY_FLAGS),
-        SendMessageOperation(response_raw, _EMPTY_FLAGS),
-    )
+    cdef tuple send_ops
+    if servicer_context._metadata_sent:
```

Comment:
I'm a bit lost with this conditional is this fixing a bug that existed with the current code? so having to send yes or yes always the initial metadata if this hasn't been sent previously?

Reply:
Yes. It looks like there was a bug. Good fix.